test/clj/rems/test_pdf.clj

```diff
@@ -125,41 +128,41 @@
-                [[:paragraph pdf/field-style "Text field"]
-                 [:paragraph "pdf test"]]
-                [[:paragraph pdf/field-style "Text area"]
-                 [:paragraph "pdf test"]]
-                [[:paragraph pdf/header-field-style "Header"]
-                 [:paragraph ""]]
-                [[:paragraph pdf/field-style "Date field"]
-                 [:paragraph "2002-03-04"]]
-                [[:paragraph pdf/field-style "Email field"]
-                 [:paragraph "user@example.com"]]
-                [[:paragraph pdf/field-style "Attachment"]
-                 [:paragraph "attachment.pdf, picture.png"]]
-                [[:paragraph pdf/field-style "Option list. Choose the first option to reveal a new field."]
-                 [:paragraph "First option"]]
-                [[:paragraph pdf/field-style "Conditional field. Shown only if first option is selected above."]
-                 [:paragraph "pdf test"]]
-                [[:paragraph pdf/field-style "Multi-select list"]
-                 [:paragraph "First option"]]
+              [[[:heading pdf/heading-style "Form"]
+                [[[:paragraph pdf/label-field-style
+                   "This form demonstrates all possible field types. (This text itself is a label field.)"]
+                  [:paragraph ""]]
+                 [[:paragraph pdf/field-style "Application title field"]
+                  [:paragraph "pdf test"]]
+                 [[:paragraph pdf/field-style "Text field"]
+                  [:paragraph "pdf test"]]
+                 [[:paragraph pdf/field-style "Text area"]
+                  [:paragraph "pdf test"]]
+                 [[:paragraph pdf/header-field-style "Header"]
+                  [:paragraph ""]]
+                 [[:paragraph pdf/field-style "Date field"]
+                  [:paragraph "2002-03-04"]]
+                 [[:paragraph pdf/field-style "Email field"]
+                  [:paragraph "user@example.com"]]
+                 [[:paragraph pdf/field-style "Attachment"]
+                  [:paragraph "attachment.pdf, picture.png"]]
+                 [[:paragraph pdf/field-style "Option list. Choose the first option to reveal a new field."]
+                  [:paragraph "First option"]]
                 [[:paragraph pdf/field-style "Table"]
                  [:paragraph
                   [:table {:header ["First" "Second"]}
                    ["pdf test" "pdf test"]
                    ["pdf test" "pdf test"]]]]
-                [[:paragraph pdf/label-field-style "The following field types can have a max length."]
-                 [:paragraph ""]]
-                [[:paragraph pdf/field-style "Text field with max length"]
-                 [:paragraph "pdf test"]]
-                [[:paragraph pdf/field-style "Text area with max length"]
-                 [:paragraph "pdf test"]]]]
+                 [[:paragraph pdf/field-style "Conditional field. Shown only if first option is selected above."]
```

Comment:
you probably mixed up the order of the fields when merging this: table should go after multi-select (see test failure)

Reply:
I guess this is the test failure, I fixed the header and then the merge has a problem.